.jenkins/pytorch/win-test.sh

```diff
@@ -17,29 +19,6 @@ if [ ! -z "$(ls $TMP_DIR/ci_scripts/*)" ]; then
     rm $TMP_DIR/ci_scripts/*
 fi
 
-cat >$TMP_DIR/ci_scripts/download_image.py << EOL
-
-import os
-import sys
-import boto3
-import botocore
-
-IMAGE_COMMIT_TAG = os.getenv('IMAGE_COMMIT_TAG')
-
-session = boto3.session.Session()
-s3 = session.resource('s3')
-BUCKET_NAME = 'ossci-windows-build'
-KEY = 'pytorch/'+IMAGE_COMMIT_TAG+'.7z'
```

Comment:
one benefit of separating these scripts is that `flake8` now checks the code.  For example, here I had to pad the `+` operator with spaces to satisfy `flake8`.